astropy/io/fits/diff.py

```diff
@@ -290,7 +306,17 @@ def __init__(self, a, b, ignore_keywords=[], ignore_comments=[],
         self.ignore_fields = set(k.upper() for k in ignore_fields)
 
         self.numdiffs = numdiffs
-        self.tolerance = tolerance
+        self.rtol = rtol
+        self.atol = atol
+
+        if tolerance is not None:  # This should be removed in the next astropy version
+            warnings.warn('"tolerance" was '
```

Comment:
This is just a PEP-8-nitpick but that seems like odd-indentation to me. Feel free to keep it as is but I would prefer:

```
        warnings.warn(
            '"tolerance" was deprecated in version 2.0 and will be removed in '
            'a future version. Use argument "rtol" instead.',
            AstropyDeprecationWarning)
```

because the hanging indentation is the same for all strings and parameters here. Same for the other occurances of this warning.